js/ui/camera.js

```diff
@@ -16,7 +16,8 @@ var Point = require('point-geometry');
  * @property {Array} center Longitude and latitude (passed as `[lng, lat]`)
  * @property {number} zoom Map zoom level
  * @property {number} bearing Map rotation bearing in degrees counter-clockwise from north
- * @property {number} pitch The angle at which the camera is looking at the ground
+ * @property {number} pitch The angle in degrees at which the camera is looking at the ground
+ * @property {Array} around If zooming, the zoom center (defaults to map center, passed as `[lng, lat]`)
```

Comment:
The full type here is `{Array<number>|LngLat}`. It's probably a good idea to add a `@typedef {Array<number>|LngLat} LngLatish` definition where we can document this precisely and talk about coordinate order in gl-js (https://github.com/mapbox/mapbox-gl-js/issues/739), and use that everywhere that a `LngLat` or array may be passed.